### PR TITLE
APM 303442 cleanup azure logs GitHub documentation after publishing Dynatrace docs

### PR DESCRIPTION
**Pull request by dtwwolinski**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -26,86 +23,31 @@ You need to deploy azure-log-forwarder in each region you want to pull logs from
...
-wget https://github.com/dynatrace-oss/dynatrace-azure-log-forwarder/releases/latest/download/dynatrace-azure-log-forwarder.zip
-``` 
-To deploy new version of azure-log-forwarder run:
-```  
-az webapp deployment source config-zip -g <resource_group> -n <app_name> --src <zip_file_path>
-```
 
-### Uninstall Dynatrace Log Forwarder
-Deployment script tags all created resources with `LogsForwarderDeployment = {DEPLOYMENT_NAME}` tag. If you want to uninstall Dynatrace Log Forwarder, in Azure Portal navigate to Resource Group used for installation, filter resources by tag, and delete resources.
+##Issues
```
Comment (pgutkowski) [inline readme.md:50]: Maybe I'm missing something - what's the purpose of having this empty section?

File: readme.md [hunk 2]
```diff
@@ -26,86 +23,31 @@ You need to deploy azure-log-forwarder in each region you want to pull logs from
 ![Architecture](./img/architecture.png)
 
 
-## Deployment
-There are two deployment options:
-
-Deployment can be run from Azure Portal Cloud Shell (Bash) or from any machine with Azure CLI and bash shell (Linux or Windows WSL) installed.
+# Deployment
+Please refer to the instructions:  
+   [Dytantrace log forwarder Documentation](https://www.dynatrace.com/support/help/shortlink/azure-log-fwd)
```
Comment (mswiatkowska) [inline readme.md:28]: Dytantrace -> Dynatrace

File: readme.md [hunk 3]
```diff
@@ -26,86 +23,31 @@ You need to deploy azure-log-forwarder in each region you want to pull logs from
...
-To deploy new version of azure-log-forwarder run:
-```  
-az webapp deployment source config-zip -g <resource_group> -n <app_name> --src <zip_file_path>
-```
 
-### Uninstall Dynatrace Log Forwarder
-Deployment script tags all created resources with `LogsForwarderDeployment = {DEPLOYMENT_NAME}` tag. If you want to uninstall Dynatrace Log Forwarder, in Azure Portal navigate to Resource Group used for installation, filter resources by tag, and delete resources.
+##Issues 
 
 #### Virtual Network cleanup
```
Comment (mswiatkowska) [inline readme.md:52]: I would remove it from github doc, it's already in Dynatrace documentation in Troubleshoot section.

Comment (mswiatkowska) [reply readme.md:52]: + it's worth to add Pricing section like in GCP doc (part with log ingest)

File: readme.md (hunk 2 above)
Comment (dtwwolinski) [reply readme.md:28]: Nice catch

File: readme.md (hunk 1 above)
Comment (dtwwolinski) [reply readme.md:50]: The issues section refers to the _`Virtual Network cleanup`_ section.

File: readme.md (hunk 3 above)
Comment (dtwwolinski) [reply readme.md]: Added

**Review by mswiatkowska — APPROVED** [review]

**Pull request merged**